projects/phone/src/lib/masks/phone/utils/generate-phone-mask.ts

```diff
@@ -0,0 +1,27 @@
+import {MaskExpression} from '@maskito/core';
+
+import {TEMPLATE_FILLER} from '../constants';
+
+export function generatePhoneMask({
+    value,
+    template,
+    prefix,
+}: {
+    value: string;
+    template: string;
+    prefix: string;
+}): MaskExpression {
+    return [
+        ...prefix.split(''),
```

Comment:
```suggestion
        ...prefix,
```

```
[...'abc'] // ['a', 'b', 'c']
```

Reply:
It requires  `"downlevelIteration": true` in ts-config. I added